Document/0x03-Overview.md

```diff
@@ -1,61 +1,78 @@
+
+Given the vast amount of mobile app frameworks available it would be impossible to cover all of them exhaustively. Therefore, we focus on _native_ apps on each operating system. However, the same techniques are also useful when dealing with web or hybrid apps (ultimately, no matter the framework, every app is based on native components).
+
+## Navigating the OWASP MASTG
+
+The MASTG contains descriptions of all requirements specified in the MASVS. The MASTG contains the following main sections:
+
+1. The [General Testing Guide](0x04a-Mobile-App-Taxonomy.md) contains a mobile app security testing methodology and general vulnerability analysis techniques as they apply to mobile app security. It also contains additional technical test cases that are OS-independent, such as authentication and session management, network communications, and cryptography.
+
+2. The [Android Testing Guide](0x05a-Platform-Overview.md) covers mobile security testing for the Android platform, including security basics, security test cases, reverse engineering techniques and prevention, and tampering techniques and prevention.
+
+3. The [iOS Testing Guide](0x06a-Platform-Overview.md) covers mobile security testing for the iOS platform, including an overview of the iOS OS, security testing, reverse engineering techniques and prevention, and tampering techniques and prevention.
+
+## How Security Personnel Should Address Mobile Security Testing
+
+Many mobile app penetration testers have a background in network and web app penetration testing, a quality that is valuable for mobile app testing. Almost every mobile app talks to a backend service, and those services are prone to the same types of attacks we are familiar with in web apps on desktop machines. Mobile apps have a smaller attack surface and therefore have more security against injection and similar attacks. Instead, the MASTG prioritizes data protection on the device and the network to increase mobile security.
 
-Many mobile app penetration testers have a background in network and web app penetration testing, a quality that is valuable for mobile app testing. Almost every mobile app talks to a backend service, and those services are prone to the same types of attacks we are familiar with in web apps on desktop machines. Mobile apps differ in that there is a smaller attack surface and therefore more security against injection and similar attacks. Instead, we must prioritize data protection on the device and the network to increase mobile security.
+## OWASP MASVS Overview: Key Areas in Mobile Application Security
 
-Let's discuss the key areas in mobile app security.
+This overview discusses how the MASVS defines and describes the key areas of mobile security:
 
-### Data Storage and Privacy (MASVS-STORAGE)
+[Data Storage and Privacy](#datastorage)
```

Comment:
```suggestion
[Data Storage and Privacy](#masvs-storage-data-storage-and-privacy)
```

Reply:
and so on

Reply:
I understand now. Just used the wrong reference online. Beg pardon.